ui/src/Queues/QueueMessagesList.test.tsx

```diff
@@ -0,0 +1,118 @@
+import React from "react";
+import { render, screen } from "@testing-library/react";
+import QueueMessagesList from "./QueueMessagesList";
+import { QueueMessage } from "./QueueMessageData";
+import { SnackbarProvider } from "../context/SnackbarContext";
+
+const mockUpdateMessageExpandedState = jest.fn();
+const mockOnRefreshMessages = jest.fn();
+
+const renderWithSnackbarProvider = (component: React.ReactElement) => {
```

Comment:
I see this helper declared 3 times. Could it be extracted to a `tests/utils.ts` so it's reused?